src/log.js

```diff
@@ -202,7 +215,7 @@ export class Log {
   error(tag, var_args) {
     const error = this.error_.apply(this, arguments);
     if (error) {
-      this.win.setTimeout(() => {throw /** @type {!Error} */ (error);});
+      reportError(error);
```

Comment:
sgtm. i don't think its a problem. just different expectation for dev that's all